README.md

```diff
@@ -1,49 +1,182 @@
+
+If you encounter any errors installing packages, try installing them manually via the terminal with no version requirements.
+
+##### Run Configuration
+
+In the menu bar, select `Run > Edit Configuration`.
+
+Click on the `+` symbol to add a new configuration and select Python. Configure as follows:
+
+- Name: Give your configuration any name
+- Script Path: select the script you want to run e.g. `manage.py` or `/app/github_public_repo/estimate_csv_creator.py`.
+- Parameters: `run`
+- Environment variables: Contact a team member for the complete list of environment variables
+- Python interpreter: select the python virtual environment you configured above
+- Interpreter options: leave blank
+- Working directory: This should automatically fill based on the `Script path` value
+- Add content roots to PYTHONPATH. Check this box
+- Add source roots to PYTHONPATH. Check this box
+
+Click OK.
+
+You should now be ready to run your script!
+
+#### PyCharm on Windows
+
+Note that this method can work using either native Windows or WSL. However, if you are using PyCharm, it is strongly recommended that you clone the repository within your **Windows user profile** and **NOT your WSL user profile**. PyCharm WSL compatibility is only available with [PyCharm Pro Edition](https://www.jetbrains.com/help/pycharm/using-wsl-as-a-remote-interpreter.html) and the workarounds for the Community Edition are not documented here. 
+
+##### Anaconda
+
+Install Anaconda [https://docs.anaconda.com/anaconda/install/windows/](https://docs.anaconda.com/anaconda/install/windows/)
+
+##### Creating virtual environment
+
+In PyCharm, open iit-backend using `File > Open`. Click *Cancel* if prompted to create a virtual environment.
+
+Create a virtualenv interpreter manually [https://www.jetbrains.com/help/pycharm/project-interpreter.html#3b6542ac](https://www.jetbrains.com/help/pycharm/project-interpreter.html#3b6542ac).
+
+When adding your python interpreter, select *Conda environment* in the lefthand side. Make sure *New environment* is selected. Give *Location* a memorable name e.g. iit-backend. Click *OK* twice to return to the main PyCharm window.
+
+##### Installing Required Packages
+
+Click *install requirements* from the PyCharm prompt or run `conda install --file requirements.txt` in the terminal. This will install some of the packages using the `conda` package manager.
+
+Many packages will fail to install using `conda`, this is expected behaviour. To install the rest of the packages, use `pip` instead of `conda`. To do this, run `pip install -r requirements.txt`.
+
+It's likely that you'll run into issues installing the `Fiona` package. To install it, run `conda install fiona` in the terminal.
```

Comment:
Yeah this one's annoying. Conda can't install "Fiona" (upper case "F"), which is what's in the requirements. But it can install "fiona", which I think is a distribution of Fiona specifically for conda. By install fiona, it installs Fiona, but python on windows can't install Fiona directly. Because we have Fiona in our requirements.txt, we have to explicitly tell conda to install fiona.